Gem/Code/Source/Components/NetworkSimplePlayerCameraComponent.cpp

```diff
@@ -45,7 +56,10 @@ namespace MultiplayerSample
 
     void NetworkSimplePlayerCameraComponentController::OnDeactivate([[maybe_unused]] Multiplayer::EntityIsMigrating entityIsMigrating)
     {
-        AZ::TickBus::Handler::BusDisconnect();
+        if(AZ::TickBus::Handler::BusIsConnected())
+        { 
+            AZ::TickBus::Handler::BusDisconnect();
+        }
```

Comment:
Last I checked, BusDisconnect is already safe so the check shouldn't be needed.

Reply:
oh yeah I can't find it now but I seem to recall there was an annoying warning message that got printed if you used it when not connected